src/components/engines/cms/modules/media/MediaActions.tsx

```
@@ -42,48 +42,48 @@ export function MediaActions({ fileUrl, s3Key, onDelete, horizontal = false }: M
   };
 
   return horizontal ? (
-    <div className="flex justify-end space-x-2" onClick={(e) => e.stopPropagation()}>
+    <div className="flex justify-end space-x-2 relative z-[9999]" onClick={(e) => e.stopPropagation()}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)